lib/electric_slide/agent_strategy/fixed_priority.rb

```diff
@@ -30,7 +30,8 @@ def available_agent_summary
       end
 
       def checkout_agent
-        _, agents = @priorities.detect do |priority, agents|
+        priorities = @priorities.sort.to_h
```

Comment:
I don't think you need to cast this to a Hash at all, it should work fine without.